_data/accessibility-tests/header.yml

```diff
@@ -0,0 +1,243 @@
+title: Header
+component_status: pass
+test_items:
+# General tests
+  - summary:
+    summary_additional:
+    test_status:
+    test_type: general
+    version_tested:
+    wcag_criterion:
+  - summary: Header content is consistent in mobile views.
+    summary_additional: |
+      When you view the page on a mobile device, 
+      the header is visible and works in both portrait and landscape settings.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.4
+  - summary: Text in header meets color contrast requirements.
+    summary_additional: |
+      When you use a color contrast checker to view the header,
+      contrast between the text color and background color is at least 4.5:1.
+    test_status: 3.8.1
+    test_type: general
+    version_tested: pass
+    wcag_criterion: 1.4.3
+  - summary: Header remains visible and functional when screen is magnified. 
+    summary_additional: |
+      When you zoom in to 400%, 
+      you can still see and use the header menu and navigation, 
+      and it doesn't overlap with or get cut off by any other content.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Search icon in header meets color contrast requirements.
+    summary_additional: |
+      When you use a color contrast checker to view the search icon, 
+      contrast between the icon and adjacent colors is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.11
+  - summary: Current page link has a distinct style.
+    summary_additional: |
+      When viewing the navigation link list, 
+      the current page has distinct styling that gives clear visual indication of where you are in the site.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.8
+  - summary: Header appears consistently.
+    summary_additional: |
+      The header appears in the same location on every page of your government website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.3
+  - summary: All clickable elements touch target size meet height and width requirements.
+    summary_additional: |
+      The size and spacing of the touch target are at least 24px wide and 24px tall.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.5.8
```

Comment:
@RachelCorsino 
**issue:** This criterion needs to be added to [wcag-details.html](https://github.com/uswds/uswds-site/blob/main/_includes/accessibility-tests/wcag-details.html). Can you make this update?

Reply:
@amyleadem - I'm not seeing a 2.5.8 version on the [Web Content Accessibility Guidelines (WCAG) 2.1 page](https://www.w3.org/TR/WCAG21)?

Reply:
Interesting! Looks like it is actually a WCAG 2.2 criterion: https://www.w3.org/TR/WCAG22/#target-size-minimum

We have one other 2.2 item in wcag-details.html: 2.4.13. We should model the 2.5.8 text on the entry for 2.4.13 (basically, include "(WCAG 2.2)" after the criterion name.

Reply:
Resolved in [f174869](https://github.com/uswds/uswds-site/pull/2761/commits/f1748690500e0bb7b483e686014be259f57066d4)